hybridesmbench/eval/_diags/sanity_checks_sum/__init__.py

```diff
@@ -0,0 +1,266 @@
+"""Run sanity checks diagnostic."""
+
+import warnings
+from typing import Any
+
+import iris
+import yaml
+from esmvalcore.preprocessor import (
+    anomalies,
+    area_statistics,
+    convert_units,
+    regrid,
+)
+from esmvaltool.diag_scripts.monitor.multi_datasets import MultiDatasets
+from iris import Constraint
+from iris.cube import Cube
+from loguru import logger
+
+from hybridesmbench._utils import (
+    get_timerange,
+)
+from hybridesmbench.eval._diags import ESMValToolDiagnostic
+from hybridesmbench.eval._loaders import Loader
+from hybridesmbench.exceptions import (
+    HybridESMBenchException,
+    HybridESMBenchWarning,
+)
+
+
+class SanityChecksDiagnostic(ESMValToolDiagnostic):
```

Comment:
```suggestion
class SanityChecksDiagnostic(ESMValToolDiagnostic):
```
You're calling both sanity checks SanitChecksDiagnostic as a class, should this also be mean/sum?